iidm/iidm-api/src/main/java/com/powsybl/iidm/network/OverloadManagementSystemAdder.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Olivier Perrin {@literal <olivier.perrin at rte-france.com>}
+ */
+public interface OverloadManagementSystemAdder extends IdentifiableAdder<OverloadManagementSystem, OverloadManagementSystemAdder> {
+
+    interface TrippingAdder<I extends TrippingAdder<I>> {
+        /**
+         * Set the unique key of the tripping.
+         * <p>This key is unique for a single overload management system, but it can be reused
+         * for another overload management system).</p>
+         * @param key the tripping key
+         */
+        I setKey(String key);
+
+        /**
+         * Set the name of the tripping.
+         * <p>This name is facultative. It is used only for reporting purposes.</p>
+         * @param name the name of the tripping
+         */
+        I setName(String name);
+
+        /**
+         * Change the state of the tripping.
+         * @param enabled <code>true</code> to enable it
+         */
+        I setEnabled(boolean enabled);
+
+        /**
+         * Set the minimum acceptable current value (in A).
+         * @param lowLimit the minimum current value
+         */
+        I setLowLimit(Double lowLimit);
+
+        /**
+         * Set the maximum acceptable current value (in A).
+         * @param highLimit the maximum current value
+         */
+        I setHighLimit(Double highLimit);
+
+        I setOpenAction(boolean open);
+
+        OverloadManagementSystem.Tripping.Type getType();
```

Comment:
This getter is used in the adder implementation to determine which type of tripping to create (see `OverloadManagementSystemAdderImpl.createTripping(AbstractTrippingAdderImpl<?>, String, Set<String>)`.

Since custom implementations may also need to read this attribute, it seems better to expose it in the interface.